spec/features/admin_user_create_events_spec.rb

```diff
@@ -1,38 +1,52 @@
 require 'rails_helper'
 
-RSpec.describe 'Admin panel: Events creation', type: :system do
+RSpec.describe 'Admin panel:', type: :system do
   include Features::AdminUserHelpers
   let(:admin_user) { build_stubbed(:admin_user) }
 
   before :each do
     sign_in_to_admin_panel(email: admin_user.email, password: admin_user.password)
   end
 
-  context 'when submitting event form with valid params' do
-    let(:event_params) { attributes_for(:event) }
+  context 'on the admin homepage' do
+    context 'has an ability to go to events index page' do
```

Comment:
You said in the context that admin _'has an ability to go to events index page'_ but 
then you check that he _'Signed in successfully.'_

Here I don't expect sign in check, but I expect something like this
```ruby
context 'on the admin homepage' do
  before do
    sign_in_to_admin_panel(email: admin_user.email, password: admin_user.password)	   
  end

  context 'has an ability to go to events index page' do
    click_link 'Events'
 
    expect(page).to have_title 'Events'
  end
```